File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewModelMyListWithMultiRowSelectionRow.cs
/// <filename>
///     ListViews_ViewModelMyListWithMultiRowSelectionRow.cs
/// </filename>
namespace widgetassertions
{
    public class ListViews_ViewModelMyListWithMultiRowSelectionRow
    {
        private string HeaderLabelText;

        public virtual string getHeaderLabelText()
        {
            return this.HeaderLabelText;
        }

        private int RowIndex;

        public virtual int getRowIndex()
        {
            return this.RowIndex;
        }
    }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewModelMyListWithMultiRowSelectionRow.cpp
#include "ListViews_ViewModelMyListWithMultiRowSelectionRow.h"
#include <string>

namespace widgetassertions
{
  std::string ListViews_ViewModelMyListWithMultiRowSelectionRow::getHeaderLabelText()
  {
    return this->HeaderLabelText;
  }
  int ListViews_ViewModelMyListWithMultiRowSelectionRow::getRowIndex()
  {
    return this->RowIndex;
  }
}
